core/templates/services/i18n-language-code.service.spec.ts

```diff
@@ -18,12 +18,14 @@
 
 import { EventEmitter } from '@angular/core';
 
-import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+import { I18nLanguageCodeService, TranslationKeyType } from 'services/i18n-language-code.service';
 import { Subscription } from 'rxjs';
 
 describe('I18nLanguageCodeService', () => {
   const i18nLanguageCodeService = new I18nLanguageCodeService();
   let languageCode: string = '';
+  let translationKey: string = '';
+  let isTranslationKeyValid: boolean;
```

Comment:
Variable name should start with a verb.

```suggestion
  let translationKeyIsValid: boolean;
```

Reply:
Done!